File: banda/JCZhang-Banda/JCZhang-Banda-JAVA/src/main/java/models/Guitarist.java
package models;


import interfaces.IGuitarist;

public class Guitarist extends Musician implements IGuitarist {

    String guitarType;
    String name;
    int yearsOfExperience;
    static double salary = (Musician.salary * 1.35);

    public Guitarist(String name, int yearsOfExperience, String GuitarType) {
        super(name, yearsOfExperience);
        this.name = name;
        this.yearsOfExperience = yearsOfExperience;
        this.guitarType = GuitarType;
    }

    @Override
    public void playGuitar() {
        System.out.println("Im a guitarist and im playing guitar");
    }
}

File: banda/JCZhang-Banda/JCZhang-Banda_KOTLIN/src/main/kotlin/models/Guitarist.kt
package models

import interfaces.IGuitarist

class Guitarist(override var name: String?, override var yearsOfExperience: Int, var guitarType: String) :
    Musician(name, yearsOfExperience), IGuitarist {
    override fun playGuitar() {
        println("Im a guitarist and im playing guitar")
    }

    companion object {
        var salary: Double = Musician.salary * 1.35
    }
}